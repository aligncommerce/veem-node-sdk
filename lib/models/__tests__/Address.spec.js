import Address from 'models/Address'

describe('Address Model', () => {
  it('should match the snapshot', () => {
    expect(Address.schema).toMatchSnapshot()
  })

  it('should define a valid schema', () => {
    expect(Address.schema).to.be.validJsonSchema
  })
})
