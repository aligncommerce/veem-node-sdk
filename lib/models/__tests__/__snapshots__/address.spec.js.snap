// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address Model should match the snapshot 1`] = `
Object {
  "properties": Object {
    "city": Object {
      "type": "string",
    },
    "line1": Object {
      "type": "string",
    },
    "line2": Object {
      "type": "string",
    },
    "postalCode": Object {
      "type": "string",
    },
    "stateProvince": Object {
      "type": "string",
    },
  },
  "title": "Address",
  "type": "object",
}
`;
